RELEASING.md

```diff
@@ -1,42 +1,38 @@
+  
+### Branch: `release-prepare`
+The `release-prepare` branch is initially created from the latest `release/vn.0.x` of the previous major release. It serves the same purpose as `main` except for the next planned Cosmos Hub Upgrade. As soon as the upgrade has taken place, it is merged into main and won't exist until the next upgrade is upcoming. During the period that `main` is the currently running network version and a new major release is being prepared, `release-prepare` is targeted for PRs that contain bug-fixes/features/improvements for that release. There may be some PRs that target `main` because they are relevant to the current network, but are also relevant to the upcoming upgrade. These should be cherry picked into `release-prepare` before being subsequently cherry picked into the respective `release/vn.0.x` that is being used for the next upgrade. The `release-prepare` branch aims to be a clean starting point for the subsequent `release` branch. Therefore, only after the binary built from `release-prepare` branch passed dev-testnet action, will this branch be used in a subsequent `release` branch. 
+
+The `release-prepare` branch will become `main` after the relevant Cosmos Hub Network Upgrade successfully takes place.
+
+## Release Procedure
+
+### Minor & Patch Releases
+
+TODO
+
+### Major Release
+
+For a new major release, `m`, after a previous release, `n`, start on `release/vn.0.x` and checkout a new branch called `upgradename-main`(e.g. `theta-main`). This will be the `release-prepare` branch referenced above going forward until the network upgrades with the new Major Release. This branch should act like `main` in that it is the target for new PRs that come in with code meant for the upcoming release. Some PRs that target the real `main` will be relevant to this `upgradename-main`, and should similarly be cherry picked to this branch. Before beginning the intitial release candidate process, this branch should pass the `dev-testnet` tests. Once passing, checkout a new branch, `release/vm.0.x` off  `release/vn.0.x`. From `release/vm.0.x` checkout two branched: `release/vm.0.0-rc0` and `rc0/vm.0.0-rc0`. Rebase `upgradename-main` into `rc0/vm.0.0-rc0`. Now open a PR from `rc0/vm.0.0-rc0` against `release/vn.0.0-rc0`. This PR is important as it's the last time code reviewers can see all changes going into the release at once. Last minute changes can go directly into `rc0/vm.0.0-rc0` as well as updaing the CHANGELOG.md. Once the PR has been merged, the `rc0/vm.0.0-rc` branch can be deleted. The `release/vm.0.0-rc0` can be tagged and the release can be published as a pre-release.
```

Comment:
"From `release/vm.0.x` checkout two branched: `release/vm.0.0-rc0` and `rc0/vm.0.0-rc0`."


`rc0/vm.0.0-rc0` should be checkout out from  `release/vm.0.0-rc0` , because `rc0/vm.0.0-rc0` is like a temporal buffer place to make commits going to  `release/vm.0.0-rc0`  will be clean. for example, the changelog of the release date will be changed at `rc0/vm.0.0-rc0` branch before merging to `release/vm.0.0-rc0` .

Reply:
"Rebase `upgradename-main` into `rc0/vm.0.0-rc0`. Now open a PR from `rc0/vm.0.0-rc0` against `release/vn.0.0-rc0`. "

just merge or cherry pick from `upgradename-main` to `rc0/vm.0.0-rc0` is enough ?
`rc0/vm.0.0-rc0` contains:
- commits from `upgradename-main 
-  some updates of changelog.